quiz.rb

```diff
@@ -7,7 +7,8 @@
 # it should return false if age is less than minimum_age
 
 def is_of_age? age, minimum_age
-  # your code goes here
+  minimum_age >= 21
```

Comment:
You have the right idea, but the age of the person and the minimum age are both parameters or the method, so you would do ` age.to_i >= minimum_age.to_i`